packages/suite-web/e2e/tests/suite/passphrase-input.test.ts

```diff
@@ -9,13 +9,20 @@ describe('Passphrase', () => {
         cy.task('setupEmu', { mnemonic: 'all all all all all all all all all all all all' });
         cy.task('startBridge');
 
-        cy.viewport(1440, 2560).resetDb();
-        cy.prefixedVisit('/');
+        cy.viewport(1980, 1080).resetDb();
+        cy.visit('/');
         cy.passThroughInitialRun();
+        cy.discoveryShouldFinish();
     });
 
     // TODO: there is a problem with clearing our password input element -> cypress deletes only last char with {selectAll}{backspace} and totally ignores .clear() command
```

Comment:
I see, [it](https://github.com/trezor/trezor-suite/pull/13980/files#r1732151647) is because of this right? we still have custom handling of input change so this comment is probably still valid https://github.com/trezor/trezor-suite/blob/develop/packages/connect-ui/src/components/Passphrase/PassphraseTypeCard.tsx#L218